src/pages/storage/forms/StoragePoolForm.tsx

```diff
@@ -75,6 +76,7 @@ export interface StoragePoolFormValues {
   pure_gateway?: string;
   pure_gateway_verify?: string;
   pure_mode?: string;
+  sourcePerClusterMember?: ClusterSpecificValues;
```

Comment:
Please move the `source: string` next to this field. The fields were not properly sorted before.